### DISCUSSION TITLE
cc2021 GPU acceleration

### DISCUSSION THREAD

**Discussion by FoxStevenHY** [LinSoftWin/Photoshop-CC2022-Linux#96] [Q&A] [answered]
Hey, Hey MiMillieuh!
Thank you for your work!!!
I am now able to use photoshop on my ArchLinux! 
Thank-you!!!!!!!!!

But I reached some problems with GPU acceleration.
Everything worked just fine if I turned on the GPU acceleration, except for the liquify function.
I couldn't open the liquify tools in the Filter tools tabs.
Every-things will freeze when I use liquify tools.
I am wondering if is it my Linux distribution problem or something..

agian
Thanks MiMillieuh!!

**Comment by MiMillieuh** [comment, marked as answer]
> Hey, Hey MiMillieuh! Thank you for your work!!! I am now able to use photoshop on my ArchLinux! Thank-you!!!!!!!!!
> 
> But I reached some problems with GPU acceleration. Everything worked just fine if I turned on the GPU acceleration, except for the liquify function. I couldn't open the liquify tools in the Filter tools tabs. Every-things will freeze when I use liquify tools. I am wondering if is it my Linux distribution problem or something..
> 
> agian Thanks MiMillieuh!!

Liquify is know to have issue with GPU acceleration on Linux. if you want to use Liquify, you can disable the GPU acceleration and it will "partially work" I think some functions aren't working without GPU acceleration and so not working on Linux.
